tests/contracts/root_chain/test_start_standard_exit.py

```diff
@@ -0,0 +1,62 @@
+import pytest
```

Comment:
I think we don't have a test for a failure of exiting utxo that is already exited.